core/nodepool/config/templates/stack-template.json

```diff
@@ -361,6 +368,7 @@
                   "Action": [
                     "autoscaling:DescribeAutoScalingGroups",
```

Comment:
On larger clusters you'd have large controller nodes too :) Anyway, I see the point and I guess adding this support isn't much of the work